inline_scan.sh

```diff
@@ -522,46 +520,38 @@ get_scan_result_pdf_by_digest() {
 }
 
 save_and_copy_images() {
-    # Save all image files to /tmp and copy to created container
-    for image in "${SCAN_IMAGES[@]-}"; do
-        local base_image_name="${image##*/}"
-        echo "Saving ${image} for local analysis"
-        local save_file_name="${base_image_name}.tar"
-        if [[ ! "${image}" =~ [:]+ ]]; then
-            save_file_name="${base_image_name}:latest.tar"
-        fi
+    local base_image_name=$(echo ${FULLTAG} | rev | cut -d '/' -f 1 | rev )
+    echo "Saving ${base_image_name} for local analysis"
+    save_file_name="${base_image_name}.tar"
 
-        IMAGE_FILES+=("$save_file_name")
+    if [[ "${v_flag-""}" ]]; then
+        mkdir -p ${VOLUME_PATH}
+        local save_file_path="${VOLUME_PATH}/${save_file_name}"
 
-        if [[ "${v_flag-""}" ]]; then
-            mkdir -p ${VOLUME_PATH}
-            local save_file_path="${VOLUME_PATH}/${save_file_name}"
-        else
-            mkdir -p /tmp/sysdig
-            local save_file_path="/tmp/sysdig/${save_file_name}"
-        fi
+        FILES_TO_CLEANUP+=("${VOLUME_PATH}/image-analysis-archive.tgz")
+        FILES_TO_CLEANUP+=("${VOLUME_PATH}/${save_file_name}")
+        # Note: adding this too because Anchore internally renames the file. See: https://github.com/anchore/ci-tools/blob/v0.7.2/scripts/image_analysis.sh#L64
```

Comment:
see comment above. I see that it's not fun to keep track of what the engine adds and any update to the engine or corner cases we didn't consider could create additional files